cpp/cmake/rapids_config.cmake

```diff
@@ -11,6 +11,9 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 # =============================================================================
+set(rapids-cmake-repo vuule/rapids-cmake)
+set(rapids-cmake-branch test-nvcomp-4.0)
+
```

Comment:
```suggestion
```

Remeber to remove before PR merge

Reply:
Note (more for me than anything): this can be reverted when https://github.com/rapidsai/rapids-cmake/pull/633 is merged.

Reply:
Done.